src/mds/MDSRank.cc

```diff
@@ -2760,7 +2765,12 @@ void MDSRankDispatcher::handle_asok_command(
 	}));
     return;
   } else if (command == "scrub abort") {
-    if (whoami != 0) {
+    if (!is_active()) {
+      *css << "MDS is not active";
+      r = -CEPHFS_EINVAL;
+      goto out;
+    }
```

Comment:
a question: is it a good practice to an asok cmd if the mds is not active? could there be any implications? im thinking maybe we allow running mds asok cmds only if mds is active? does it make sense?

Reply:
we can technically run (some) command for a standby-replay mds.